server/controllers/user.js

```diff
@@ -0,0 +1,86 @@
+import models from '../models';
+import jwt from 'jsonwebtoken';
+const user = models.User;
+
+const UserController = {
+  addUser(req, res) {
+    return user 
+        .create({
+            lastName: req.body.lastName,
+            firstName: req.body.firstName,
+            username: req.body.username,
+            password: req.body.password,
+            email: req.body.email,
+            active: req.body.active
+        })
+        .then(() => {
+            res.send ({
+                status: 'Success',
+                message: "User successfully added"
+            });
+        })
+        .catch(error => res.status(400).send({
+            status: false,
+            message: error
+        }));
+  },
+  login(req, res) {
+    return user
+      .findOne({
+        where: { 
+            username: req.body.username,
+            password: req.body.password
+         },
+      })
+      .then((user) => {
+          if (!user) {
+              return res.status(404).send({
+                  message: 'User not found! Please create an account'
+              });
+          }
+          else{
+            
+            const token = jwt.sign({
+                id: user.id }, 
+                process.env.SIGNATURE);
+
+            return res.json({
+            success: true,
+            message: 'Token generated!',
```

Comment:
kindly change `message` to `Login successful` as it better explains this response